tests/dataset_runner.py

```diff
@@ -205,17 +205,17 @@ def _count_primary_bundles_analysis_workflows_and_results_bundles(self):
             self._batch_count_analysis_workflows_by_project_shortname()
         if self._results_bundles_count() < self.expected_bundle_count:
             self._count_results_bundles()
-        Progress.report("  bundles: primary: {0}/{1}, results: {2}/{3} \n  workflows: running: {4}/{5}, \
-                        succeeded: {6}/{7}, failed: {8}/{9}".format(
+        Progress.report("  primary bundles: {0}/{1} \n  workflows: running: {2}/{3}, \
```

Comment:
Minor nit: There are some missing commas here

Reply:
bundles and workflows are separated by `\n` so I think it's fine to not have commas?